src/util/get-date.util.ts

```diff
@@ -0,0 +1,12 @@
+import { Reference } from "../model/reference.model";
+
+/**
+ * Given a reference as parameter, returns a JS Date object
+ */
+export function getDate (ref: Reference): Date {
+    let raw = ref.issued["date-parts"][0];
+    let res: Date = new Date();
+    if (raw[0] !== null) res.setFullYear(raw[0]);       // Year
+    if (raw[1] !== null) res.setMonth(raw[1]);          // Month
+    return res;
```

Comment:
Shouldn't you assign the day too?

Reply:
For the day, I didn't find a method `setDay()` but it can be set using the constructor, so I can do some workarounds.

Reply:
I believe that would be `setDate()`